external/nginx/src/os/unix/ngx_alloc.h

```diff
@@ -9,10 +9,11 @@
 #define _NGX_ALLOC_H_INCLUDED_
 
 
-#include <ia2.h>
 #include <ngx_config.h>
 #include <ngx_core.h>
 
+#include <ia2.h>
+
```

Comment:
Why does #include order in this commit matter?

Reply:
When IA2 is disabled we now don't set _GNU_SOURCE but we include headers. Once a glibc header has been included, the configuration of *_SOURCE is then fixed. We could skip including the headers if IA2 is disabled?